### ISSUE TITLE
Possible to add to System Tray?

### ISSUE THREAD

**Issue by trgeiger** [jsalatas/plasma-pstate#7]
Is there any possibility that this could live inside the KDE System Tray, much like how the Redshift widget can be enabled there?

**Comment by jsalatas** [comment]
This was my original intention. The "problem" with the tray is that the widget cannot expand to its own size, but the tray imposes some kind of fixed size for it: Try for example clicking on several plasma widgets on your system tray and you will see that all of these have the same fixed size.

So I guess I should either introduce a scrollview, which looks really ugly or somehow redesign the whole user interface to fit in that size :\

**Comment by jsalatas** [comment]
> A few of the standard parts of the System Tray have scrollable parts, so maybe if some of the interface was smaller it wouldn't be so bad to scroll? Not sure.

Yeah! Maybe a less fancy interface, eg dropdown boxes instead of big fancy buttons with icons. I'll give it a try... :)

> I just thought it might be nice if the icon used for the widget could be changed manually by the user?

Sure!

**Comment by jsalatas** [comment]
Could you please test the latest code? You should be able to change the icon.
